pkg/landscape/app/notify.hoon

```diff
@@ -289,10 +319,12 @@
       =.  clients.u.entry  (~(put by clients.u.entry) who `sid)
       this(provider-state (~(put by provider-state) service u.entry))
     ::
-        [%remove-binding *]
-      `this
+        [%remove-binding *]  `this
     ::
         [%send-notification *]
+      ?>  ?=(%iris -.sign-arvo)
+      ?>  ?=(%http-response +<.sign-arvo)
+      ?>  ?=(%finished -.client-response.sign-arvo)
       `this
```

Comment:
Do we not care about handling non-200 responses here? (Or `%cancel`, though iirc those never happen in practice rn. Might still be nice to prep for them existing though.)

Reply:
This didn't actually cause non-200 response to break in my testing, but idk, happy to change

Reply:
Well they won't break, because they will come back as `%finished`, but you might want to recognize, say, a `500`, or some legible api-level error message, and retry submitting the notification, or things like that?

Reply:
Given the way that notifications works, I'm not sure retrying makes sense, but yeah, we should probably at least log this